scan/main.py

```diff
@@ -1,38 +1,31 @@
-# main.py
-
+import sys
 import os
 
-from crewai import Crew, Process
-from dotenv import load_dotenv
-from langchain_google_genai import ChatGoogleGenerativeAI
+# Ensure the project root is in the PYTHONPATH
+sys.path.append(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
I think this could lead to issues. Instead if you want to add it to path I would install the package with [pipx](https://github.com/pypa/pipx)

Reply:
Thanks for the pipx suggestion! I initially went with modifying the PYTHONPATH because it got things running. Considering pipx is typically for global installs, how about we switch to using a virtual environment for managing our local dependencies more cleanly? What steps would you suggest for setting it up if we go that route?

Reply:
So we are already using virtual environments, you just don't see it with the way you are using poetry (its default). 

For development I wouldn't use pipx, only for installing the final package. For that its easy if you publish to you, or you can also pipx install from the GitHub repo.

Reply:
I updated the main.py file [most recent commit](https://github.com/iLevyTate/SCAN/pull/13/commits/2f5571c1b3fa2ca1e860ffc79c88dc47f75c7bf2) and it worked?

Reply:
This looks good.